_activities/gsoc.md

```diff
@@ -24,47 +24,120 @@ To answer these questions, particle physicists build software to simulate and an
+
+The 2021 GSoC edition extended the student eligibility criteria. Due to the large number of candidates in 2020 and an expected increase for this year, the selection process in our organization will be split in two phases. All candidates will have to pass pre-selection evaluation tests prepared by mentors, demonstrating the skills needed for the respective projects. Only the successful candidates will be able to engage in a personalized exchange with mentors on a given project idea, and their proposals will be evaluated for the final student selection. The detailed [timeline](/activities/gsoc.html#timeline) for these phases is shown below.
 
 ### For Students
 
-If you are interested in the GSoC program, please start by having a look at our [guidelines for students](/gsoc/students-guideline.html).
+If you are interested in the the HSF projects, please start by having a look at our [guideline for students](/gsoc/students-guideline.html).
+
+New projects for HSF GSoC in 2021 will be published by **February 19**. Meanwhile you can take a look at last year's HSF GSoC [projects](/gsoc/2020/index.html#projects-in-2020) since many of those may propose subjects also this year. The student application period will begin on **March 29**, but discussions about student project ideas with mentors will take place starting with **March 9**, when the list of accepted Organizations is announced. You are encouraged to monitor this website in order to get acquainted with our projects and be prepared for contacting their mentors.
 
-New projects for HSF GSoC in 2020 will be published in February. The application period will open in mid-March, but discussions about application ideas with mentors will take place the month before. You can monitor our website in order to get acquainted with the projects you are interested in and to be prepared for submitting an application. Meanwhile you may want to take a look at last year's HSF GSoC [projects](/gsoc/2019/index.html) since many of those may propose subjects also this year.
+Please avoid having the first contact with the mentors of HSF projects before **March 9** or after **March 20**! You are encouraged to take the time to read our project proposals until **March 9**, focusing on only one or two projects that attract your interest. Once you have identified those, you should e-mail the respective mentors, attaching your CV and describing the motivation of your choice. The mentors will first propose a test of their choice, waiting for your solution. Note that the test you are given is private, your solution should be personal and the response time is part of the evaluation. Mentors will let you know the test results, and if you passed they will start discussing and helping you develop a project idea. The student application period is **March 29 - April 13**, when you will have to write and review your proposal with help from your mentors.
```

Comment:
Should we add a justification for this? "Due to the high volume of applicants..."

Reply:
Added in the paragraph before: "Due to the high number of applicants for CERN-HSF projects, the application period procedure become more formal this year, so please try to follow our guideline."